bogota-abril-2019/final_project/marvin-cely/tenis-match/src/components/PlayersPageComponent/FormNewPlayer.js

```diff
@@ -0,0 +1,75 @@
+import React from '../../../node_modules/react';
+import '../../css/FormNewPlayer.css';
+import { observer } from '../../../node_modules/mobx-react';
+import {CountryOptionComponent} from '../SmallPieceComponent';
+import utils from '../../utils/utils';
+
+const FormNewPlayer = observer(
+  class FormNewPlayer extends React.Component {
+    constructor(props) {
+      super(props);
+
+      this.state = {
+        fullNameForm: 'Nombre completo',
```

Comment:
Those all values are static content, you shouldn't set them in the state component, you should use an external file for static data or using in variables in render methods.

Reply:
static content was passed to data-content and some title component are props